### PR TITLE
ci: use setup-cpp to install llvm

### PR DESCRIPTION
**Pull request by aminya**
This fixes Windows CI failures

Also tested on libclang here:
https://github.com/atilaneves/libclang/pull/34

### CODE REVIEW

**Comment by atilaneves** [comment]
Red CI.

**Comment by aminya** [comment]
> Red CI.

All green now.

File: README.md [hunk 1]
```diff
@@ -252,6 +252,8 @@ After the instructions for your OS (see below), you can use this commands to run
 dub run dpp -- yoursourcefilenamehere.dpp
 ```
 
+Note: to reproduce the environment of dpp's CI, you can run [setup-cpp](https://github.com/aminya/setup-cpp) with `--llvm=11.0.0` and `--vcvarsall=true`. This will set up LLVM 11.0.0 and the proper environment variables.
```
Comment (atilaneves) [inline README.md:255]: Shouldn't this be in the Windows section?

Comment (aminya) [reply README.md:255]: setup-cpp is cross platform.

**Review by atilaneves — APPROVED** [review]

**Pull request merged**